compose/cli/colors.py

```diff
@@ -41,9 +41,9 @@ def make_color_fn(code):
 
 
 def rainbow():
-    cs = ['cyan', 'yellow', 'green', 'magenta', 'red', 'blue',
+    cs = ['cyan', 'yellow', 'green', 'magenta', 'blue',
           'intense_cyan', 'intense_yellow', 'intense_green',
-          'intense_magenta', 'intense_red', 'intense_blue']
+          'intense_magenta', 'intense_blue']
```

Comment:
Would be nice to remove `intense_blue` as well since it's not easily readable with a black background. Or considering replacing it with something else.